src/utils/util.js

```diff
@@ -412,6 +412,33 @@ const _Dom = {
   createElement(tagName: string): any {
     return document.createElement(tagName);
   },
+  /**
+   * Loads an external style sheet asynchronously.
+   * @param {string} url - The css url to load.
+   * @return {Promise} - The loading promise.
+   * @public
+   */
+  loadStyleSheetAsync(url: string): Promise<*> {
+    return new Promise((resolve, reject) => {
+      let r = false,
+        t = document.getElementsByTagName('link')[0],
+        s = this.createElement('link');
+      s.type = 'text/css';
+      s.rel = 'stylesheet';
+      s.href = url;
+      s.async = true;
+      s.onload = s.onreadystatechange = function() {
+        if (!r && (!this.readyState || this.readyState === 'complete')) {
+          r = true;
+          resolve(this);
```

Comment:
why are you passing `this`?

Reply:
It gives a ref to the link element that was added to the DOM. Actually I don't think of a use case that someone will need the reference for that... I can remove it

Reply:
so you can set the promise type to return <null>